.gitignore

```diff
@@ -180,6 +180,7 @@ java/**/pom.xml
 # python virtual env
 venv
 .venv
+.venv*
```

Comment:
```suggestion
```

Reply:
This would try to stage all virtual environments that are not of name `.venv`  exactly.